packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -1,23 +1,20 @@
 <template>
-	<div class="kt-button-wrapper" style="display: contents">
-		<button
-			ref="triggerRef"
-			v-bind="$attrs"
-			:class="mainClasses"
-			role="button"
-			:type="isSubmit ? 'submit' : 'button'"
-			@click="handleClick"
-		>
-			<i v-if="isLoading" class="kt-circle-loading" />
-			<i v-if="hasIconLeft" class="yoco" v-text="icon" />
-			<span v-if="hasSlot">
-				<slot />
-			</span>
-			<span v-else-if="label !== null" v-text="label" />
-			<i v-if="hasIconRight" class="yoco" v-text="icon" />
-		</button>
-		<div v-if="showHelpText" ref="contentRef" v-text="helpText" />
-	</div>
+	<button
+		ref="helpTextTriggerRef"
+		:class="mainClasses"
+		role="button"
+		:type="isSubmit ? 'submit' : 'button'"
```

Comment:
from what I understood only if we have "input of type submit" we would need the "role="button"" so maybe that's why the role was there


@FlorianWendelborn do you have an opinion here?

Reply:
This should definitely stay as-is as it’s a fix for a really ugly and unexpected default HTML behavior where the first button inside a form is automatically assumed to be the submit button. This means that a form like this

```vue
<template>
  <form>
    <input v-model="modelValue" />
    <KtButton label="Delete" @click="deleteMe" />
  </form>
</template>
```

would end-up deleting itself if the user edits the `input` and hits `enter`

See-also: #191 #190 #179